src/actions/downloadMetadataFromUri.ts

```diff
@@ -0,0 +1,20 @@
+import { DomainMetadata, IPFSGatewayUri } from "../types";
+import { makeApiCall } from "../api/actions/helpers";
+
+export const downloadMetadataFromUri = async (
```

Comment:
nit: rename this `getDomainMetadataFromUri` for consistency with `getDomainMetadata`,